pkg/localmetrics/localmetrics.go

```diff
@@ -85,21 +85,24 @@ func NewMetricsCollector(store cache.Cache) *MetricsCollector {
 			Help:        "Report the size of account pool cr",
 			ConstLabels: prometheus.Labels{"name": operatorName},
 		}, []string{"namespace", "pool_name"}),
-		accountReadyDuration: prometheus.NewSummary(prometheus.SummaryOpts{
+		accountReadyDuration: prometheus.NewHistogram(prometheus.HistogramOpts{
 			Name:        "aws_account_operator_account_ready_duration_seconds",
 			Help:        "The duration for account cr to get ready",
 			ConstLabels: prometheus.Labels{"name": operatorName},
+			Buckets:     []float64{1, 3, 5, 10, 15, 20, 30},
```

Comment:
@fahlmant @rogbas Please review buckets

Reply:
@fahlmant Do we need to add a ccs label for account cr?

Reply:
And the buckets need to update, I saw some metrics with ready time > 100s.

And there is one metric with ready time = 17655s. Seems there is a bug.